test/e2e/scheduling/limit_range.go

```diff
@@ -222,6 +226,89 @@ var _ = SIGDescribe("LimitRange", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should ensure that a limitRange can be listed, patched and deleted by collection", func() {
+
+		lrClient := f.ClientSet.CoreV1().LimitRanges(f.Namespace.Name)
+		lrName := "e2e-limitrange-" + utilrand.String(5)
+		createdLabel := map[string]string{lrName: "created"}
+		createdLabelSelector := labels.SelectorFromSet(createdLabel).String()
+		patchedLabelSelector := lrName + "=patched"
+
+		min := getResourceList("50m", "100Mi", "100Gi")
+		max := getResourceList("500m", "500Mi", "500Gi")
+		defaultLimit := getResourceList("500m", "500Mi", "500Gi")
+		defaultRequest := getResourceList("100m", "200Mi", "200Gi")
+		maxLimitRequestRatio := v1.ResourceList{}
+		value := strconv.Itoa(time.Now().Nanosecond()) + string(uuid.NewUUID())
+
+		limitRange := &v1.LimitRange{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: lrName,
+				Labels: map[string]string{
+					"time": value,
+					lrName: createdLabel[lrName],
+				},
+			},
+			Spec: v1.LimitRangeSpec{
+				Limits: []v1.LimitRangeItem{
+					{
+						Type:                 v1.LimitTypeContainer,
+						Min:                  min,
+						Max:                  max,
+						Default:              defaultLimit,
+						DefaultRequest:       defaultRequest,
+						MaxLimitRequestRatio: maxLimitRequestRatio,
+					},
+				},
+			},
+		}
+
+		ginkgo.By(fmt.Sprintf("Creating LimitRange %q", lrName))
+		limitRange, err := lrClient.Create(context.TODO(), limitRange, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "Failed to create limitRange %q", lrName)
+
+		// Listing across all namespaces to verify api endpoint: listCoreV1LimitRangeForAllNamespaces
+		ginkgo.By(fmt.Sprintf("Listing all LimitRanges with label %q", createdLabelSelector))
+		limitRangeList, err := f.ClientSet.CoreV1().LimitRanges("").List(context.TODO(), metav1.ListOptions{LabelSelector: createdLabelSelector})
```

Comment:
If listing all namespaces, it's probably worth having a LR in at least 2 namespaces.